selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java

```diff
@@ -67,85 +67,83 @@
           + "30:    if (numGuessByUser != null && numGuessByUser.equals(secretNum)) {\n"
           + "34:    else if (numGuessByUser != null) {";
 
-  @Inject private FindUsages findUsages;
-  @Inject private TestWorkspace workspace;
   @Inject private Ide ide;
-  @Inject private ProjectExplorer projectExplorer;
-  @Inject private CodenvyEditor editor;
   @Inject private Menu menu;
   @Inject private Loader loader;
-  @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private Events events;
+  @Inject private CodenvyEditor editor;
+  @Inject private FindUsages findUsages;
+  @Inject private TestWorkspace workspace;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
 
   @BeforeClass
   public void setUp() throws Exception {
     URL resource = getClass().getResource("/projects/guess-project");
     testProjectServiceClient.importProject(
         workspace.getId(), Paths.get(resource.toURI()), PROJECT_NAME, MAVEN_SPRING);
+
     ide.open(workspace);
+    ide.waitOpenedWorkspaceIsReadyToUse();
   }
 
   @Test
   public void checkFindUsagesBaseOperation() {
-    ide.waitOpenedWorkspaceIsReadyToUse();
     projectExplorer.waitItem(PROJECT_NAME);
     projectExplorer.quickExpandWithJavaScript();
     projectExplorer.openItemByVisibleNameInExplorer("AppController.java");
 
-    // Check basic operations of the 'find usages' panel
+    // Check basic operations of the FindUsages panel
     editor.selectTabByName("AppController");
     editor.goToCursorPositionVisible(27, 17);
     menu.runCommand(ASSISTANT, FIND_USAGES);
     loader.waitOnClosed();
     findUsages.waitFindUsagesPanelIsOpen();
-    events.clickEventLogBtn();
-    findUsages.waitFindUsagesPanelIsClosed();
-    findUsages.clickFindUsagesIcon();
-    findUsages.waitFindUsagesPanelIsOpen();
     findUsages.waitExpectedTextInFindUsagesPanel(EXPECTED_TEXT);
     findUsages.waitSelectedElementInFindUsagesPanel("numGuessByUser");
 
-    // Check basic operations of the 'find usages' panel
-    editor.selectTabByName("AppController");
-    editor.goToCursorPositionVisible(27, 17);
-    menu.runCommand(ASSISTANT, FIND_USAGES);
-    loader.waitOnClosed();
+    // Switch to the Events panel and check that the FindUsages panel is not visible
+    findUsages.waitFindUsagesPanelIsOpen();
+    events.clickEventLogBtn();
+    events.waitOpened();
+    findUsages.waitFindUsagesPanelIsClosed();
+
+    // Switch to the FindUsages panel and check its expected content
+    findUsages.clickFindUsagesIcon();
     findUsages.waitFindUsagesPanelIsOpen();
     findUsages.waitExpectedTextInFindUsagesPanel(EXPECTED_TEXT);
     findUsages.waitSelectedElementInFindUsagesPanel("numGuessByUser");
 
-    // Check nodes in the 'find usages' panel by 'double click' and click on the icon node
+    // Check nodes in the FindUsages panel by 'double click' and click on the icon node
```

Comment:
Code will be slightly clearer if we put comment `click on the icon node` at the proper place in the code below.